pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -625,6 +637,114 @@ func (pc *PodController) initializeSecondaryInterfaceStore() error {
 	return nil
 }
 
+// initializeSRIOVSecondaryInterfaceStore restores secondary interfaceStore for SR-IOV interfaces
+// when agent restarts. It will get the Pod info from the store of primary interfaces, and check
+// the NetworkStatus annotation of a Pod, then restore the SR-IOV interfaces based on the NetworkAttachmentDefinition
+// name and device's pci_address in the NetworkStatus.
+func (pc *PodController) initializeSRIOVSecondaryInterfaceStore() error {
+	knownInterfaces := pc.primaryInterfaceStore.GetInterfacesByType(interfacestore.ContainerInterface)
+	for _, ifconf := range knownInterfaces {
+		podNamespace := ifconf.ContainerInterfaceConfig.PodNamespace
+		podName := ifconf.ContainerInterfaceConfig.PodName
+		pod, err := pc.kubeClient.CoreV1().Pods(podNamespace).Get(context.TODO(), podName, metav1.GetOptions{})
+		if err != nil {
+			klog.ErrorS(err, "Failed to get Pod", "Pod", klog.KRef(podNamespace, podName))
+			continue
+		}
+		objRef := klog.KObj(pod)
+		_, found := checkForPodSecondaryNetworkAttachment(pod)
+		if !found {
+			klog.V(2).InfoS("Pod does not have a NetworkAttachmentDefinition", "Pod", objRef)
+			continue
+		}
+		netStatus, err := netdefutils.GetNetworkStatus(pod)
+		if err != nil {
+			klog.V(2).ErrorS(err, "Failed to get NetworkStatus for Pod", "Pod", objRef)
```

Comment:
Is the error ignorable? Why is it V(2)?

Reply:
Removed V(2). Considering we checked the NAD of Pod before this step, a network status annotation should be expected here, so remove it to have the log by default log level.